telemetry/influxdb/src/main/scala/io/buoyant/telemetry/influxdb/InfluxDbTelemeter.scala

```diff
@@ -0,0 +1,123 @@
+package io.buoyant.telemetry.influxdb
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.http.{MediaType, Request}
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.finagle.tracing.NullTracer
+import com.twitter.util.{Awaitable, Closable, Future}
+import io.buoyant.admin.Admin
+import io.buoyant.telemetry.{Metric, MetricsTree, Telemeter}
+
+/**
+ * This telemeter exposes metrics data in the InfluxDb LINE format, served on
+ * an admin endpoint.  It does not provide a StatsReceiver or Tracer.  It reads
+ * histogram summaries directly off of the MetricsTree and assumes that stats
+ * are being snapshotted at some appropriate interval.
+ */
+class InfluxDbTelemeter(metrics: MetricsTree) extends Telemeter with Admin.WithHandlers {
+
+  private[influxdb] val handler = Service.mk { request: Request =>
+    val host = request.host.getOrElse("none")
+    val response = request.response
+    response.mediaType = MediaType.Txt
+    val sb = new StringBuilder()
+    writeMetrics(metrics, sb, Nil, Seq(("host", host)))
+    response.contentString = sb.toString
+    Future.value(response)
+  }
+
+  val adminHandlers: Seq[Admin.Handler] = Seq(
+    Admin.Handler("/admin/metrics/influxdb", handler)
+  )
+
+  val stats = NullStatsReceiver
+  def tracer = NullTracer
+  def run(): Closable with Awaitable[Unit] = Telemeter.nopRun
+
+  private[this] val disallowedChars = "[^a-zA-Z0-9:]".r
+  private[this] def escapeKey(key: String) = disallowedChars.replaceAllIn(key, "_")
+
+  private[this] def formatLabels(labels: Seq[(String, String)]): String =
+    if (labels.nonEmpty) {
+      labels.sortBy(_._1).map {
+        case (k, v) =>
+          s"""$k=$v"""
+      }.mkString(",")
+    } else {
+      ""
+    }
+
+  private[this] def labelExists(labels: Seq[(String, String)], name: String) =
+    labels.exists(_._1 == name)
```

Comment:
This will allocate a function for each call to `labelExists`.  Not sure if there's a super great way to avoid this, best I could come up with is:

```
val first = ((String, String)) => String = _._1
private[this] def labelExists(labels: Seq[(String, String)], name: String) = labels.toIterator.map(first).contains(name)
```

Unfortunately that's not super readable, but I THINK it saves on allocation which might matter depending on how often this endpoint is going to be scraped.